app/routes/events/utils.js

```diff
@@ -184,3 +186,11 @@ export const hasPaid = (chargeStatus: string) =>
   paymentSuccessMappings[chargeStatus];
 
 export const addStripeFee = (price: number) => Math.ceil(price * 1.012 + 1.8);
+
+// Good
```

Comment:
Remove this comment

Reply:
I thought we discussed this onsite. This comment is the cornerstone of this whole function. People need to be able to see that this is a good function and well-tested function.